app/presenters/track.rb

```diff
@@ -0,0 +1,39 @@
+module ExercismWeb
+  module Presenters
+    class Track
+
+      def initialize(track_id)
+        @track_id = track_id
+        @track = Trackler.tracks[track_id]
+      end
+
+      def method_missing(method, *args)
+        @track.send(method, *args)
```

Comment:
Should we check if `@track.respond_to?` before doing a `@track.send`? If it doesn't respond to the method we can invoke `super`.